### ISSUE TITLE
Peer dependency warning on theo@^8

### ISSUE THREAD

**Issue by matthewdavidson** [salesforce-ux/gulp-theo#6]
At present, `npm install --save-dev gulp-theo theo` produces the following warning: 
```
npm WARN gulp-theo@2.0.0 requires a peer of theo@^7.0.1 but none is installed. You must install peer dependencies yourself.
```

**Comment by khawkins98** [comment]
Saw the fix for this in the source, would be great if a `2.0.1` could be released to clean up install logs :)

**Comment by zkuzmic** [comment]
I second this. The fix is there! But no one can use it until you publish a patch. :(

**Comment by zkuzmic** [comment]
FYI, for anyone who needs a workaround until then, you can point to the latest commit in github instead of the latest npm version in your `package.json`:

`"gulp-theo": "https://github.com/salesforce-ux/gulp-theo.git#bda9a82c9ae96952ec35d46cc09de583fada4ff1",`

**Comment by zahnster** [comment]
Thanks for reporting this issue and for your patience in awaiting its resolution. We have updated the dependencies and released v2.0.1 to resolve this issue.
